pymc/logprob/transforms.py

```diff
@@ -940,6 +1030,20 @@ def log_jac_det(self, value, *inputs):
             return value
 
 
+class SigmoidTransform(RVTransform):
```

Comment:
For the sigmoid we can use the `1 / 1 + exp(-x)` equivalence, which PyMC should be able to then derive as well. PyTensor has many rewrites to stabilize sigmoid forms so no need to worry about it at this point

Reply:
Yeah that makes sense. Would it also make sense to tie in the `LogOddsTransform` into similar a rewrite, `pt.log(inp/1-inp)`?

Reply:
That one requires a special logic because inp shows up twice in the graph. PyMC should actually refuse to convert that graph (I am curious if it actually does). 

It should require its own specialized measurable rewrite.